### PR TITLE
Add root method

### PR DESCRIPTION
**Pull request by cfabianski**

### CODE REVIEW

File: lib/ltree_hierarchy/hierarchy.rb [hunk 1]
```diff
@@ -146,6 +146,10 @@ def depth # 1-based, for compatibility with ltree's nlevel().
         end
       end
 
+      def root
+        ltree_scope.where("#{ltree_path_column} = subpath(?, 0, 1)", ltree_path)
```
Comment (tomekw) [inline lib/ltree_hierarchy/hierarchy.rb]: @cfabianski `.first`?

Comment (cfabianski) [inline lib/ltree_hierarchy/hierarchy.rb]: Yup

Comment (tomekw) [inline lib/ltree_hierarchy/hierarchy.rb]: Plus also update the `README`? :)

**Comment by ChuckJHardy** [comment]
:+1:

**Comment by tomekw** [comment]
:star:

**Pull request merged**